### DISCUSSION TITLE
How to control the bold fonts of headerToolbar title, weekday, day in calendar?

### DISCUSSION THREAD

**Discussion by jinhuang11** [vkurko/calendar#318] [General]
How to control the bold fonts of headerToolbar title, weekday, day in calendar? The demo on the left is showing bold font, but my code in the right is not showing as bold font. 

<img width="2171" alt="image" src="https://github.com/user-attachments/assets/2be564af-3af3-4b9b-8679-a669c82c927e">

Thank you so much for you help!

**Comment by vkurko** [comment]
This is just CSS. These places use `<h2>`, `<h4>` headings. Apparently your styles reset font weight for these tags. Just add the necessary rules for `h2.ec-title` and `h4.ec-day-head`.

**Reply by jinhuang11** [reply]
Thanks @vkurko ! Could you share some examples for how add necessary rules for h2.ec-title and h4.ec-day-head.

**Reply by vkurko** [reply]
Sure:

```css
h2.ec-title, h4.ec-day-head {
    font-weight: bold;
}
```

**Reply by jinhuang11** [reply]
Amazing!! It works perfectly now!!! Thanks you so much @vkurko !!
